docs/cli-reference/gardenadm/gardenadm_token_create.md

```diff
@@ -32,10 +32,12 @@ gardenadm token create
 ### Options
 
 ```
-  -d, --description string                        Description for the bootstrap token (default "Used for joining nodes via `gardenadm join` or connecting autonomous shoots via `gardenadm connect`")
+  -d, --description gardenadm join                Description for the bootstrap token used for gardenadm join
```

Comment:
I understand the old format, but have trouble with the new one. Why was `string` changed to `gardenadm join`? The previous variant is clearly denoting a type. I do not understand what the current version is highlighting.
Please explain.